### ISSUE TITLE
Cannot download linked imgur albums

### ISSUE THREAD

**Issue by TinyTrbl** [johanneszab/TumblThree#324]
Hello.

Im really sorry for asking, but i cannot download imgur albums from new posts (2018) on [NSFW artworks](http://giallo1972.tumblr.com/). "Download imgur" option is enabled.

- Expected result: downloaded a picture from tumblr post + downloaded a larger copy from imgur
- Actual result: downloaded a picture from tumblr post, but not from imgur

Am I doing something wrong or it is not possible?

**Comment by johanneszab** [comment]
Looks like they are linked differently again in that blog. It's probably just a line of code (adjusting the regular expression in the [Imgur parser](https://github.com/johanneszab/TumblThree/blob/master/src/TumblThree/TumblThree.Applications/Parser/ImgurParser.cs)), but for sure it could be the case that for that blog, you'll not be able to download the linked images.

 If there would be one single way to do something, things would be too easy, really, ..

**Comment by johanneszab** [comment]
Well, I've actually looked into this, and couldn't get it easily working, .. then I tested the urls in the blog, and they are all dead as well. For example:

https://imgur.com/4B2BSpz
https://imgur.com/T2FpMRQ

I'm not even sure if those have a valid format.

**Comment by johanneszab** [comment]
No, I believe you. I had similar experience when I was implementing the tumblr _raw photo detection. The current Imgur RegEx for album detection would also not detect those links above as they are in the form

https://imgur.com/4B2BSpz
https://imgur.com/T2FpMRQ

instead of

[https://imgur.com/**a**/4B2BSpz](https://imgur.com/**a**/4B2BSpz)
[https://imgur.com/**A**/T2FpMRQ](https://imgur.com/**A**/T2FpMRQ)

without the a or A after the host.
